src/hotspot/os/aix/os_aix.cpp

```diff
@@ -1016,7 +1016,7 @@ static void* dll_load_library(const char *filename, char *ebuf, int ebuflen) {
   void* result;
   const char* error_report = nullptr;
   JFR_ONLY(NativeLibraryLoadEvent load_event(filename, &result);)
-  result = Aix_dlopen(filename, dflags, &error_report);
+  result = Aix_dlopen(filename, eno, dflags, &error_report);
```

Comment:
Please switch dflags and eno
 old:    result = Aix_dlopen(filename, eno, dflags, &error_report);
 new:  result = Aix_dlopen(filename, dflags, eno, &error_report);

Reply:
ok. why are we switching the order of arguments  / parameters ? is it some order of the arguments to be processed ?